nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/ui/common/edit-parameter-dialog/edit-parameter-dialog.component.ts

```diff
@@ -163,8 +166,8 @@ export class EditParameterDialog {
         this.editParameter.next({
             parameter: {
                 name: this.editParameterForm.get('name')?.value,
-                value,
-                valueRemoved: value == '' && !empty,
+                value: value === '' && !empty ? null : value,
+                valueRemoved: value === '' && !empty,
```

Comment:
When applying the Parameter Context update, this `valueRemoved` field is getting lost and doesn't not get included in the update request. As a result, it's not possible when the user attempts to clear a field it is not removed.